sdk/formrecognizer/azure-ai-formrecognizer/swagger/README.md

```diff
@@ -20,7 +20,7 @@ autorest --java --use=C:/work/autorest.java
 
 ### Code generation settings
 ``` yaml
-input-file: ./v2.1.json
```

Comment:
There is some one changes in the file that doesn't not have in the released swagger. It is about the multiple service API version support. If you want to use the released one, you can add one transformed change in the README.

It is better to create another stand-alone PR for it. I will create one soon.

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/21794

Reply:
Just a reminder we are releasing this week. I will say we need at least 2 days of nightly green builds before we release because of how important the release is.
If the swagger transform change can't make it before that, that is ok. Priority is make sure no bugs are introduced

Reply:
There is no bugs introduced by having this changes.

Reply:
Running live tests on this PR to make sure we are not introducing any bugs https://github.com/Azure/azure-sdk-for-java/pull/21789#issuecomment-848077260